ui-admin/src/portal/dashboard/PortalDashboard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+import { Portal } from 'api/api'
+import { renderPageHeader } from 'util/pageUtils'
+
+import { StudyWidget } from './widgets/StudyWidget'
+import { WebsiteWidget } from './widgets/WebsiteWidget'
+import { TutorialsWidget } from './widgets/TutorialsWidget'
+import { MailingListWidget } from './widgets/MailingListWidget'
+import { StudyTeamWidget } from './widgets/StudyTeamWidget'
+import { StudyTrendsWidget } from './widgets/StudyTrendsWidget'
+import classNames from 'classnames'
+
+export default function PortalDashboard({ portal }: {portal: Portal}) {
```

Comment:
I love the modular design of this - let's wrap each widget in an ErrorBoundary to add extra stability

Reply:
good call!